core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -618,6 +618,7 @@ write_files:
       ETCD_ADVERTISE_CLIENT_URLS=https://$advertised_hostname:2379
       ETCD_LISTEN_PEER_URLS=https://$private_ip:2380
       ETCD_INITIAL_ADVERTISE_PEER_URLS=https://$advertised_hostname:2380" >> /var/run/coreos/etcd-environment
+      ETCD_VERSION={{.Etcd.Version}}
```

Comment:
Sorry this is confusing but given what we have so far:

```
echo "KUBE_AWS_ASSUMED_HOSTNAME=$advertised_hostname
*snip*
ETCD_INITIAL_ADVERTISE_PEER_URLS=https://$advertised_hostname:2380" >> /var/run/coreos/etcd-environment
ETCD_VERSION={{.Etcd.Version}}
```

It should be the something like the below:

```
echo "KUBE_AWS_ASSUMED_HOSTNAME=$advertised_hostname
*snip*
ETCD_INITIAL_ADVERTISE_PEER_URLS=https://$advertised_hostname:2380
ETCD_VERSION={{.Etcd.Version}}" >> /var/run/coreos/etcd-environment
```

or `ETCD_VERSION` would be just thrown away.